configs/11.0/packages/gcc/stage_1

```diff
@@ -31,8 +31,8 @@ atcfg_pre_hacks() {
 	echo "Advance-Toolchain-${at_major_internal}" > ${ATSRC_PACKAGE_WORK}/gcc/DEV-PHASE
 	# Overwrite the existing REVISION with the current SVN revision date
 	mv ${ATSRC_PACKAGE_WORK}/gcc/REVISION ${ATSRC_PACKAGE_WORK}/gcc/REVISION.back
-	sed "s/branch merged/branch, revision: ${ATSRC_PACKAGE_REV} merged/" ${ATSRC_PACKAGE_WORK}/gcc/REVISION.back \
-		 > ${ATSRC_PACKAGE_WORK}/gcc/REVISION
+	sed "s/ \([^ ][^ ]*\)]$/ ${ATSRC_PACKAGE_REV}, based upon FSF revision \1]/" \
+	        ${ATSRC_PACKAGE_WORK}/gcc/REVISION.back > ${ATSRC_PACKAGE_WORK}/gcc/REVISION
```

Comment:
I think this code needs a mechanism to detect when sed does not match anything.  This way we would catch this kind if issues earlier.
Maybe a diff between REVISION.back and REVISION?

Reply:
Does AT next also come with a gcc/REVISION or do we need to generate it?

The older AT versions (at least 9.0 and 10.0) also need this fix.